galaxy_ng/app/api/ui/urls.py

```diff
@@ -8,6 +8,7 @@
 router.register('namespaces', viewsets.NamespaceViewSet, basename='namespaces')
 router.register('my-namespaces', viewsets.MyNamespaceViewSet, basename='my-namespaces')
 router.register('collections', viewsets.CollectionViewSet, basename='collections')
+router.register('users', viewsets.UserViewSet, basename='users')
```

Comment:
How do we remove or disable the `users` endpoints when running on c.rh.com?

Reply:
This is a good question. On the UI we have a flag to toggle between on prem and cloud. I'm not a huge fan of this approach, but we were sort of forced into doing it because of some of the limitations with webpack. @cutwater do you have any more pythonic ideas?

https://github.com/ansible/ansible-hub-ui/blob/master/src/api/user.ts#L13

Reply:
Should this router be disabled for insights mode?

Reply:
I don't think that's necessary. People shouldn't be using the `_ui` api anyway.